libraries/botbuilder-dialogs-adaptive-teams/src/conditions/onTeamsTeamUnarchived.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * @module botbuilder-dialogs-adaptive-teams
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Expression, ExpressionParserInterface } from 'adaptive-expressions';
+import { Channels } from 'botbuilder';
+import { Dialog, TurnPath } from 'botbuilder-dialogs';
+import { OnConversationUpdateActivity } from 'botbuilder-dialogs-adaptive';
+
+/**
+ * Actions triggered when a Teams ConversationUpdate with channelData.eventType == 'teamUnarchived'.
+ * Note: turn.activity.channelData.Teams has team data.
+ */
+export class OnTeamsTeamUnarchived extends OnConversationUpdateActivity {
+    public static $kind = 'Teams.OnTeamUnarchived';
+
+    public constructor(actions?: Dialog[], condition?: string) {
+        super(actions, condition);
```

Comment:
If you are only invoking `super`, you can (likely) omit the constructor override entirely.